core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.ts

```diff
@@ -199,6 +202,43 @@ angular.module('oppia').directive('adminMiscTab', [
           );
         };
 
+        ctrl.getModelsRelatedToUser = function() {
+          ctrl.setStatusMessage('Getting the models related to user...');
+          $http.get(GET_MODELS_RELATED_TO_USER_HANDLER_URL, {
```

Comment:
Move all the http requests to the backend API service. (We have an active PR where we are moving all the admin-page related HTTP requests to backend API services)